src/coreclr/src/gc/gc.cpp

```diff
@@ -20,6 +20,7 @@
 #include "gcpriv.h"
 
 #define USE_INTROSORT
+#define ALLOW_REFERENCES_IN_POH
```

Comment:
this would be better in gcpriv.h which has all the feature related defines. the #define's in gc.cpp are purely implementation related, ie, you wouldn't expect someone to define or define them to enable a feature.